Src/Support/Google.Apis.Core/Util/ExponentialBackOff.cs

```diff
@@ -33,12 +33,23 @@ public class ExponentialBackOff : IBackOff
 
         /// <summary>
         /// Time span used to bound the back-off jitter.
-        /// If the value is <see cref="System.TimeSpan.Zero"/> then the generated back-off will be exactly 1, 2, 4,
+        /// If the value is <see cref="TimeSpan.Zero"/> then the generated back-off will be exactly 1, 2, 4,
         /// 8, 16, etc. seconds. A valid value is between zero and one second. The default value is 250ms, which means
         /// that the generated back-off will be [0.75-1.25]sec, [1.75-2.25]sec, [3.75-4.25]sec, and so on.
+        /// Only one of <see cref="DeltaBackOff"/> and <see cref="DeltaBackOffPercent"/> may be specified. Both may be zero,
+        /// in which case no jitter will be applied.
         /// </summary>
         public TimeSpan DeltaBackOff { get; }
 
+        /// <summary>
+        /// Percent of the current unjittered back-off used to bound the jitter.
+        /// If the value is set to 10, and the current back-off is 1000ms, then the jitter will be generated
+        /// from [-100, 100]ms and the jittered back-off will be [900,1100]ms.
```

Comment:
I'd change this to "-100ms to 100ms" or similar, to avoid anything thinking it may be markdown. (Ditto the later part.)